content/en/docs/pluggable-components/app-store.md

```diff
@@ -0,0 +1,115 @@
+
+1. In the tutorial of [Installing KubeSphere on Kubernetes](https://kubesphere-v3.netlify.app/docs/installing-on-kubernetes/introduction/overview/), you execute `kubectl apply -f` first for the file [kubesphere-installer.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/kubesphere-installer.yaml). After that, to enable App Store, create a local file cluster-configuration.yaml.
+
+```bash
+vi cluster-configuration.yaml
+```
+
+2. Copy all the content in the file [cluster-configuration.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/cluster-configuration.yaml) and paste it to the local file just created.
+3. In this local cluster-configuration.yaml file, navigate to `openpitrix` and enable App Store by changing `false` to `true` for `enabled`. Save the file after you finish.
+
+```bash
+openpitrix:
+    enabled: true # Change "false" to "true"
+```
+
+4. Execute the following command to start installation:
+
+```bash
+kubectl apply -f cluster-configuration.yaml
+```
+
+Whether you install KubeSphere on Linux or on Kubernetes, you can check the status of the component in the web console of KubeSphere after installation in **Components**.
+
+## Enable App Store after Installation
+
+1. Log in the console as `admin`. Click **Platform** at the top left corner and select **Clusters Management**.
+
+![clusters-management](https://ap3.qingstor.com/kubesphere-website/docs/20200828111130.png)
+
+2. Click **CRDs** and enter `clusterconfiguration` in the search bar. Click the result to view its detailed page.
+
+{{< notice info >}}
+
+A Custom Resource Definition (CRD) allows users to create a new type of resources without adding another API server. They can use these resources like any other native Kubernetes objects.
+
+{{</ notice >}}
+
+3. In **Resource List**, click the three dots on the right of `ks-installer` and select **Edit YAML**.
+
+![edit-yaml](https://ap3.qingstor.com/kubesphere-website/docs/20200827182002.png)
+
+4. In this yaml file, navigate to `openpitrix` and change `false` to `true` for `enabled`. After you finish, click **Update** at the bottom right corner to save the configuration.
+
+```bash
+openpitrix:
+    enabled: true # Change "false" to "true"
+```
+
+5. You can use the web kubectl to check the installation process by executing the following command:
+
+```bash
+kubectl logs -n kubesphere-system $(kubectl get pod -n kubesphere-system -l app=ks-install -o jsonpath='{.items[0].metadata.name}') -f
+```
+
+{{< notice tip >}}
+
+You can find the web kubectl tool by clicking the hammer icon at the bottom right corner of the console.
+
+{{</ notice >}}
+
+6. Check the status in **Components** or execute `kubectl get pod --all-namespaces` through kubectl to see the status of pods.
```

Comment:
Please change this part as follows:

Check the status in **Components** or execute the following command through kubectl to see the status of related pods.

```
kubectl get pod -n openpitrix-system
```

The correct running status as follows:

```
NAME                                                 READY   STATUS      RESTARTS   AGE
hyperpitrix-generate-kubeconfig-8npd2                0/2     Completed   0          19d
hyperpitrix-release-app-job-mstr6                    0/1     Completed   0          19d
openpitrix-hyperpitrix-deployment-6797497b67-2fl6g   1/1     Running     7          19d
```

Reply:
You need to add the corresponding command for checking each pluggable component.

Reply:
That's quite reasonable. The command has been added.